src/components/modules/banners/environmentMomentBanner/helpers/ophan.ts

```diff
@@ -0,0 +1,38 @@
+import { OphanComponentEvent } from '../../../../../types/OphanTypes';
+
+const OPHAN_COMPONENT_ID_READ_PLEDGE = 'environment-moment-2020-read-pledge';
+const OPHAN_COMPONENT_ID_CONTRIBUTE = 'environment-moment-2020-contribute';
+const OPHAN_COMPONENT_ID_HEAR_FROM_OUR_EDITOR = 'environment-moment-2020-hear-from-our-editor';
+const OPHAN_COMPONENT_ID_CLOSE = 'environment-moment-2020-close';
+
+export const OPHAN_COMPONENT_EVENT_READ_PLEDGE_CLICK: OphanComponentEvent = {
+    component: {
+        componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
+    },
+    id: OPHAN_COMPONENT_ID_READ_PLEDGE,
```

Comment:
Let's use `component.id` instead of this `id` for each of these.
E.g. here the component is the pledge cta

Reply:
👍